plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/client/IndexMetadata.java

```diff
@@ -89,6 +119,25 @@ public String getName()
         {
             return name;
         }
+
+        @Override
+        public int hashCode()
+        {
+            return Objects.hash(name);
+        }
+
+        @Override
+        public boolean equals(Object o)
```

Comment:
Let's implement for all types consistently

Reply:
Handled it. let me know if you know any better way to do it consistently?